src/lib.rs

```diff
@@ -192,6 +216,37 @@ impl Capturer {
     /// Get the last frame stored in `self` by `Self::capture_store_frame`,
     /// if one has ever been stored.
     pub fn get_stored_frame(&self) -> Option<&[Bgr8]> {
-        self.image.as_ref().map(|img| img.as_slice())
+        self.image.as_deref()
```

Comment:
This would have been good, except `Image` on `x11cap` doesn't implement `Deref` so it doesn't compile on Linux.

Reply:
Reverted: I assume there is no benefit for adding a gated windows version and keep the original as a gated non-windows version.
I'll probably benchmark it after fixing the issue I'm having with the dxgcap crate.